src/main/kotlin/com/cognifide/gradle/aem/deploy/tasks/DistributeTask.kt

```diff
@@ -1,4 +1,4 @@
-package com.cognifide.gradle.aem.deploy
+package com.cognifide.gradle.aem.deploy.tasks
```

Comment:
personally separating tasks doesn't give me more cleanness. Moreover I dislike it, because in other packages there are only one task for each package and they are not separated into special subpackage. I prefer to leave it as is.